c7n/resources/opsworks.py

```diff
@@ -155,6 +207,106 @@ def process_stack(self, stack):
             self.log.exception(
                 "Exception stopping stack:\n %s" % e)
 
+@OpsworkStack.action_registry.register('start')
+class StartStack(BaseAction):
+    """Action to start Opswork Stack
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: opswork-stop
+                resource: opswork-stack
+                actions:
+                  - start
+    """
+
+    schema = type_schema('start')
+    permissions = ("opsworks:StartStack",)
+
+    def process(self, stacks):
+        with self.executor_factory(max_workers=10) as w:
+            list(w.map(self.process_stack, stacks))
+
+    def process_stack(self, stack):
+        client = local_session(
+            self.manager.session_factory).client('opsworks')
+        try:
+            stack_id = stack['StackId']
+            client.start_stack(StackId=stack_id)
+        except ClientError as e:
+            self.log.exception(
+                "Exception starting stack:\n %s" % e)
```

Comment:
remove the new line character here please

Reply:
Stop and Delete actions have the same character, I've just copied this part of the code. Do you think this character should be removed from of all these parts?